example/vault/environments/local/versions.tf

```
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0.8"
```

I would align this with the .terrafor-version, since this is an example, not a reusable module.
```suggestion
  required_version = ">= 1.3.4"
```

---

I don't want to force any consumer to this version as it is not a necessity. We have tested this example already with 1.0.8 successfully. 

We don't know if people use tfenv either.